src/mavis/convert/vcf.py

```diff
@@ -193,25 +291,9 @@ def convert_record(record: VcfRecordType) -> List[Dict]:
                 }
             )
         else:
-            std_row.update(
-                {
-                    COLUMNS.break1_position_start: max(
-                        1, record.pos + info.get('CIPOS', (0, 0))[0]
-                    ),
-                    COLUMNS.break1_position_end: record.pos + info.get('CIPOS', (0, 0))[1],
-                    COLUMNS.break2_position_start: max(1, end + info.get('CIEND', (0, 0))[0]),
-                    COLUMNS.break2_position_end: end + info.get('CIEND', (0, 0))[1],
-                }
-            )
-        if std_row['break1_position_end'] == 0 and std_row['break1_position_start'] == 1:
-            # addresses cases where pos = 0 and telomeric BND alt syntax https://github.com/bcgsc/mavis/issues/294
-            std_row.update({'break1_position_end': 1})
-        if std_row['break2_position_end'] == 0 and std_row['break2_position_start'] == 1:
-            std_row.update({'break2_position_end': 1})
-
-        if 'SVTYPE' in info:
-            std_row[COLUMNS.event_type] = info['SVTYPE']
-
+            convert_imprecise_breakend(std_row, record, end)
```

Comment:
(with the throw/catch model)

```python
else:
    try:
        convert_imprecise_breakend(std_row, record, end)
    except ValueError as err:
        logging.error(f'error in processing breakend of record {record.id}: {err}')
        continue
```

In fact it would probably be even better to move the try/catch up into the "grandparent" function since this error would occur for every iteration of this loop  otherwise which is a bit inefficient